packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -378,6 +387,20 @@ export class LoggingError extends Error implements ILoggingError, Omit<IFluidErr
         }
     }
 
+    /**
+    * Determines if a given object is an instance of a LoggingError
+    * @param object - any object
+    * @returns - true if the object is an instance of a LoggingError, false if not.
+    */
+    public static typeCheck(object: unknown): object is LoggingError {
```

Comment:
Can we use the same naming/calling convention for `LoggingError.typeCheck` and `isNormalizedLoggingError`? i.e. put the second on the class as `static typeCheck` or move the first out to a bare function `isLoggingError`